modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -121,6 +149,16 @@ func (fsrv *FileServer) Provision(ctx caddy.Context) error {
 		fsrv.IndexNames = defaultIndexNames
 	}
 
+	// for show paths that are static (i.e. no placeholders), we can transform them into
+	// absolute paths before the server starts for very slight performance improvement
+	for i, s := range fsrv.Show {
```

Comment:
Similarly, can we combine this section and the next section (remove whitespace) and just have one comment that covers both?